src/applications/financial-status-report/containers/App.jsx

```diff
@@ -48,17 +47,27 @@ const App = ({
     return <ErrorMessage />;
   }
 
```

Comment:
Looks goo just need to add a no-index to this files so its not found by search engines. Here is a PR where I removed it so you can see how to add it in.

https://github.com/department-of-veterans-affairs/vets-website/pull/15460/files#diff-47002777ea4220b1566c6596b86c9de15a7cac122c9329094f5cac749eb570c0L4-L28

Reply:
yep I was wondering where that should go thanks for the heads up